tests/phpunit/elementor/modules/usage/test-module.php

```diff
@@ -47,6 +47,65 @@ class Test_Module extends Elementor_Test_Base {
 		],
 	];
 
+	static $document_mock_default_with_container = [
```

Comment:
@TzviRabinovitch Can you maybe mock the following structures as well?

- Nested elements
- A page that has two parent containers, and one of them has a child container.

WDYT?